packages/angular/stories/generated/wrappers-form.mdx

```diff
@@ -0,0 +1,138 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Wrappers/Form" />
+
+# Wrappers/Form
+
+{/*
+This file is automatically generated from wrappers-form.stories.ts
+Do not edit this file directly. Run 'yarn extract-templates' to regenerate.
+To add custom introduction content, create wrappers-form.intro.md */}
+
+For examples of the different types of forms and how to bind and read values you can look at the `Forms` section in the root of the Angular storybook.
+
+## Checkbox
```

Comment:
The description of the component is missing (also for other components in other files).